### PR TITLE
Refactor Subcommands

### PR DESCRIPTION
**Pull request by Swiizyy**
Closes: #160

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,6 +52,7 @@
 		"eslint-plugin-tsdoc": "^0.2.17",
 		"husky": "^8.0.3",
 		"lint-staged": "^13.2.0",
+		"npm-run-all": "^4.1.5",
```
Comment (nikolaischunk) [inline package.json:55]: what is this for?

Comment (Swiizyy) [reply package.json:55]: this is required for the docker config.
without this we can't start the bot :/

File: src/commands/general/config/reset.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+						value: 'announcementMessage',
+					},
+					// {
+					// 	name: 'Logs',
+					// 	value: 'logs'
+					// }
+				)
+				.setRequired(true),
+		),
+)
```
Comment (nikolaischunk) [inline src/commands/general/config/reset.ts:46]: Please outsource this into a separate file in the `lib/commands/config` file if possible

File: package.json (hunk 1 above)
Comment (Swiizyy) [reply package.json:55]: I thought we didn't need it but we do

File: package.json [hunk 3]
```diff
@@ -19,6 +20,7 @@
 		"@sapphire/plugin-scheduled-tasks": "^6.0.0",
 		"@sapphire/plugin-subcommands": "^4.0.0",
 		"@sapphire/plugin-utilities-store": "^1.0.0",
+		"@sapphire/result": "^2.6.0",
```
Comment (nikolaischunk) [inline package.json:23]: what is this plugin for?

**Review by nikolaischunk — APPROVED** [review]

**Comment by Swiizyy** [comment]
LGTM 🚀

**Pull request merged**